catroid/src/org/catrobat/catroid/ui/adapter/LookAdapter.java

```diff
@@ -59,7 +60,14 @@ public long getItemId(int position) {
 			return INVALID_ID;
 		}
 		LookData item = getItem(position);
-		return idMap.get(item);
+
+		for (Map.Entry<LookData, Integer> e : idMap.entrySet()) {
+			if (item.equals(e.getKey())) {
+				return e.getValue();
+			}
+		}
+
+		return INVALID_ID;
```

Comment:
Hmm, the implementation of LookData.hashCode is wrong, right?
https://github.com/Catrobat/Catroid/blob/dba80632feb54a4ef9dada08c46e94df57d518dd/catroid/src/org/catrobat/catroid/common/LookData.java#L84-L87